### ISSUE TITLE
Fix Announcement issue on double confirmation modal when closed without any edits

### ISSUE THREAD

**Issue by Sulaymon333** [grommet/hpe-design-system#5912]
### Link to problem

http://localhost:3000/components/layer?q=layer#double-confirmation-anatomy

### Current problem

SR does (Screen Reader ) not announce double confirmation modals message if user does not make have any unsaved edits and simply open those modal and close it immediately. 

Note: There is a [PR](https://github.com/grommet/hpe-design-system/pull/5865) up for creation of double confirmation page. It currently lives in layer page. Once the PR is merged, you can find the double confirmation modals on Side drawer, double confirmation and Layer pages

### Expected behavior or solution

Screen Reader should announce `Modal closed` or something similar to let non-sighted users be aware that the modal closed.
